repositories.bzl

```diff
@@ -61,14 +61,14 @@ def com_google_api_codegen_repositories():
         urls = ["https://github.com/bazelbuild/bazel-skylib/archive/2169ae1c374aab4a09aa90e65efe1a3aad4e279b.tar.gz"],
     )
 
-    _maybe(
-        http_archive,
-        name = "net_zlib",
-        build_file = "@com_google_protobuf//:third_party/zlib.BUILD",
-        sha256 = "c3e5e9fdd5004dcb542feda5ee4f0ff0744628baf8ed2dd5d66f8ca1197cb1a1",
-        strip_prefix = "zlib-1.2.11",
-        urls = ["https://zlib.net/zlib-1.2.11.tar.gz"],
-    )
+#    _maybe(
+#        http_archive,
```

Comment:
Why's this commented out?

Reply:
This should have been simply deleted (deleting them). The reason is a recent change in protobuf repository, where they started referrencing `zlib` as `@zlib` instead of `//external:zlib" (The commit: https://github.com/protocolbuffers/protobuf/commit/f313b9ccf7434e04d233ad2f519affc78a9e26a8). 

Since now protobuf depends on zlib differently, the dependency itself is pulled differently and in a different place (i.e. by protobuf itself)